pandas/core/arrays/datetimelike.py

```diff
@@ -258,7 +258,11 @@ def _unbox_scalar(
 
         Examples
         --------
-        >>> self._unbox_scalar(Timedelta("10s"))  # doctest: +SKIP
+        >>> obj = DatetimeLikeArrayMixin(
+        ...    OpsMixin(),
+        ...    NDArrayBackedExtensionArray(NDArrayBacked(), ExtensionArray())
+        ...    )
+        >>> obj._unbox_scalar(Timedelta("10s"))  # doctest: +SKIP
```

Comment:
this is quite hard to understand 😄 I'd suggest either using a concrete example (and removing doctest: SKIP) like
```python
>>> arr = pd.arrays.DatetimeArray(np.array(['1970-01-01'], 'datetime64[ns]'))
>>> arr._unbox_scalar(arr[0])
numpy.datetime64('1970-01-01T00:00:00.000000000')
```
, or just leaving it alone as it's a private method anyway

Reply:
Understood, I've updated the example!